app/controllers/contact_messages_controller.rb

```diff
@@ -0,0 +1,31 @@
+class ContactMessagesController < ApplicationController
+  def new
+    @contact_message = ContactMessage.new
+  end
+
+  def create
+    @contact_message = ContactMessage.new(contact_message_params)
+    if @contact_message.save
+      NewContactMessageMailer.contact_message_notification(
+        @contact_message).deliver_now
+      flash[:success] =
+        "It's good to hear from you. We will respond to your message shortly."
+      redirect_to root_path
+    else
+      flash.now[:error] =
+        "Please review your message, some information is missing."
+      render :new
+    end
+  end
+
+  private
+
+  def contact_message_params
+    params.require(:contact_message).permit(
+      :name,
+      :email,
+      :cell,
+      :message
```

Comment:
I was kidding about the length, but my main issue with alphabetizing is that it creates an artificial constraint that leads to disagreement. Typical scenarios are that you forget to alphabetize and then have to deal with signal to noise issues in PRs. Another scenario is the alphabetizing often gives the illusion that it's faster to search large lists visually rather than using your editor. In the long run I end up not caring either way, but I never alphabetize.